app/migration/mappers/training_programme_type_mapper.rb

```diff
@@ -0,0 +1,19 @@
+module Mappers
+  class TrainingProgrammeTypeMapper
```

Comment:
Minor naming thought - since it's already name spaced under `Mappers`, would it make sense to just call the class `TrainingProgramme` instead of `TrainingProgrammeTypeMapper`?